<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>921</width>
    <height>751</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="calculator">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>921</width>
      <height>751</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(16, 16, 16);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="float_2">
     <widget class="QPushButton" name="switchtoint">
      <property name="geometry">
       <rect>
        <x>740</x>
        <y>670</y>
        <width>181</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>Switch To
Integer Mode</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>921</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);
background-color: rgb(45, 45, 45);</string>
      </property>
      <property name="text">
       <string>                                         Floating Mode</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>41</width>
        <height>711</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(45, 45, 45);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTextEdit" name="binScreen_f">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>190</y>
        <width>861</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="decScreen_f">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>360</y>
        <width>861</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>150</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>Binary</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>320</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>Decimal</string>
      </property>
     </widget>
     <widget class="QPushButton" name="binToDec_f">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>240</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Convert To Dec</string>
      </property>
     </widget>
     <widget class="QPushButton" name="decToBin_f">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>410</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Convert To Bin</string>
      </property>
     </widget>
     <zorder>switchtoint</zorder>
     <zorder>label_6</zorder>
     <zorder>label_7</zorder>
     <zorder>label_8</zorder>
     <zorder>label_9</zorder>
     <zorder>binToDec_f</zorder>
     <zorder>decToBin_f</zorder>
     <zorder>binScreen_f</zorder>
     <zorder>decScreen_f</zorder>
    </widget>
    <widget class="QWidget" name="int_2">
     <widget class="QPushButton" name="switchtofloat">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>670</y>
        <width>261</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>Switch To
Floating Point Mode</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>81</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>Binary</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>240</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>Decimal</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>410</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);</string>
      </property>
      <property name="text">
       <string>Heximal</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>921</width>
        <height>41</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 85, 0);
background-color: rgb(45, 45, 45);</string>
      </property>
      <property name="text">
       <string>                                         Integer Mode</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="binScreen_i">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>110</y>
        <width>821</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="decScreen_i">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>821</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="hexScreen_i">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>450</y>
        <width>821</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>17</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="equalBin_i">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>110</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="equalDec_i">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>280</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="equalHex_i">
      <property name="geometry">
       <rect>
        <x>870</x>
        <y>450</y>
        <width>51</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>=</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dichTrai">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>670</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>SHL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dichPhai">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>670</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>SHR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="xoayTrai">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>670</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ROL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="xoayPhai">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>670</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>ROR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="binToDec_i">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>160</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Convert To Dec</string>
      </property>
     </widget>
     <widget class="QPushButton" name="binToHex_i">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>160</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Convert To Hex</string>
      </property>
     </widget>
     <widget class="QPushButton" name="decToBin_i">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>330</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Convert To Bin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="decToHex_i">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>330</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Convert To Hex</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hexToBin_i">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>500</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Convert To Bin</string>
      </property>
     </widget>
     <widget class="QPushButton" name="hexToDec_i">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>500</y>
        <width>181</width>
        <height>61</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Convert To Dec</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>41</width>
        <height>711</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(45, 45, 45);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="not_i">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>670</y>
        <width>81</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(42, 42, 42);
color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>NOT</string>
      </property>
     </widget>
     <zorder>switchtofloat</zorder>
     <zorder>xoayPhai</zorder>
     <zorder>xoayTrai</zorder>
     <zorder>dichTrai</zorder>
     <zorder>label_5</zorder>
     <zorder>equalHex_i</zorder>
     <zorder>equalDec_i</zorder>
     <zorder>equalBin_i</zorder>
     <zorder>label</zorder>
     <zorder>label_2</zorder>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>dichPhai</zorder>
     <zorder>binToDec_i</zorder>
     <zorder>binToHex_i</zorder>
     <zorder>decToBin_i</zorder>
     <zorder>decToHex_i</zorder>
     <zorder>hexToBin_i</zorder>
     <zorder>hexToDec_i</zorder>
     <zorder>binScreen_i</zorder>
     <zorder>decScreen_i</zorder>
     <zorder>hexScreen_i</zorder>
     <zorder>not_i</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
